person.rb

```diff
@@ -13,6 +17,10 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
```

Comment:
- [ ] Greate job so far 👏🏻 However according to the project requirements, you should implement decorator and run a program using the command line. Actually, I'm getting this error shown above 👇🏻 Kidly solve this 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/234436337-59f6cd38-b329-4ac2-97be-e6038fbc4fa4.png)

- You may use this code in the `person` class to check if you managed to decorate a class:
```ruby
  person = Person.new(22, 'maximilianus')
  person.correct_name
  capitalizedPerson = Capitalize.new(person)
  capitalizedPerson.correct_name
  capitalizedTrimmedPerson = ...
   ...
```

Reply:
@BenMukebo  Thanks it's fixed